packages/mobile/src/store/storeContext.ts

```diff
@@ -4,12 +4,16 @@ import type { CommonStoreContext } from 'audius-client/src/common/store'
 import { apiClient } from 'app/services/audius-api-client'
 import { audiusBackendInstance } from 'app/services/audius-backend-instance'
 import { fingerprintClient } from 'app/services/fingerprint'
-import { remoteConfigInstance } from 'app/services/remote-config'
+import {
+  getFeatureEnabled,
+  remoteConfigInstance
+} from 'app/services/remote-config'
 import { walletClient } from 'app/services/wallet-client'
 
 export const storeContext: CommonStoreContext = {
   getLocalStorageItem: async (key) => AsyncStorage.getItem(key),
   setLocalStorageItem: async (key, value) => AsyncStorage.setItem(key, value),
+  getFeatureEnabled,
```

Comment:
so can we use remoteConfigInstance to call getFeatureEnabled, or are they separate things?

Reply:
They are separate things right now. The `getFeatureFlag` helper checks local storage for the feature flag overrides, but I think I can consolidate it

Reply:
Actually we do need both, the one in remoteConfig is used by the feature flag modal to show the default state. The helper is used by actual features to check if they're enabled accounting for overrides. I could return the one from audiusBackendInstance out and update the usages to use that one, but it seems like it makes more sense at the top level of the storeContext. wdyt?

Reply:
yeah im good with top level, but was just wondering why it seemed like we had duplicate things. maybe when everything is sorted, we take a loot at context and see if we can consolidate? things like getLocalStorageItem vs localStorage, is another example